<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaCliente</class>
 <widget class="QDialog" name="TelaCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastrar Cliente</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/rec/img/business_application_addmale_useradd_insert_add_user_client_2312.png</normaloff>:/rec/img/business_application_addmale_useradd_insert_add_user_client_2312.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>51</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Nome">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>73</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Email">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CPF">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>73</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>51</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CPF</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CNH">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>170</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefone</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Telefone">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>120</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>150</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CNH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Salvar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>  Salvar</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/img/devices_floppydisk_dispositivos_691.png</normaloff>:/rec/img/devices_floppydisk_dispositivos_691.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>29</width>
     <height>28</height>
    </size>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_imagem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_InserirFoto">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserir Foto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">qproperty-icon:url(:/rec/img/reply-arrow-filled.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Nome</tabstop>
  <tabstop>lineEdit_CPF</tabstop>
  <tabstop>lineEdit_Email</tabstop>
  <tabstop>lineEdit_Telefone</tabstop>
  <tabstop>lineEdit_CNH</tabstop>
  <tabstop>pushButton_InserirFoto</tabstop>
  <tabstop>pushButton_Salvar</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
